hessian_test/dup_struct_name_test.go

```diff
@@ -181,3 +181,9 @@ func checkResponseBody(t *testing.T, decodedResponse *hessian.Response, h *hessi
 	out, _ := hessian.EnsureInterface(hessian.UnpackPtrValue(hessian.EnsurePackValue(decodedResponse.RspObj)), nil)
 	assert.Equal(t, in, out)
 }
+
+func TestGetGoName(t *testing.T) {
+	assert.Equal(t, "time.Time", hessian.GetGoName(time.Now()))
+	assert.Equal(t, "github.com/apache/dubbo-go-hessian2/hessian_test_test/hessian_test.CaseZ", hessian.GetGoName(&CaseZ{}))
+	assert.Equal(t, "github.com/apache/dubbo-go-hessian2/hessian_test/hessian_test/hessian_test.CaseZ", hessian.GetGoName(&dupclass.CaseZ{}))
+}
```

Comment:
Why test GetGoName twice? I think the test cases could be removed, and use the use cases at `pojo_test.go`.